### PR TITLE
Латышев Алексей Сергеевич, группа 2539

### PR DESCRIPTION
**Pull request by alex-700**
Rss-reader. Second part.

### CODE REVIEW

**Comment by thevery** [comment]
apk?

**Comment by alex-700** [comment]
I'm sorry. Apk will be added a minute later.

**Comment by thevery** [comment]
I've built apk myself

File: app/src/main/java/alex700/com/arssreader/FeedFetcherService.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    public static void loadChannel(Context context, String channel) {
+        context.startService(new Intent(context, FeedFetcherService.class).putExtra(CHANNEL_NAME, channel));
+    }
+
+
+    public static boolean isLoading(String channel) {
+        return tasks.contains(channel);
+    }
+
+    public static void setHandler(Handler handler) {
```
Comment (thevery) [inline app/src/main/java/alex700/com/arssreader/FeedFetcherService.java:44]: you should not use `static` here

**Comment by thevery** [comment]
ok

12-1 => 6

**Pull request closed without merging**